cmd/cephcsi.go

```diff
@@ -52,6 +56,12 @@ var (
 	// cephfs related flags
 	volumeMounter = flag.String("volumemounter", "", "default volume mounter (possible options are 'kernel', 'fuse')")
 	mountCacheDir = flag.String("mountcachedir", "", "mount info cache save dir")
+
+	// livenes related flags
+	livenessport = flag.Int("liveness-port", 8080, "TCP port for liveness requests")
+	livenesspath = flag.String("liveness-path", "/metrics", "path of prometheus endpoint where metrics will be available")
+	pollTime     = flag.Duration("poll-time", time.Second*60, "time interval in seconds between each poll")
+	probeTimeout = flag.Duration("probe-timeout", time.Second*3, "probe timeout in seconds")
```

Comment:
can we call this as just `timeout` as other sidecar containers are doing?

Reply:
currently, we don't use `-` in the flags. do we need to deprecate the current flags and add `-` to it (some mechanism to support old flags) (not in this PR)